jsonrpc/src/schemas/portal.json

```diff
@@ -38,5 +38,21 @@
     "title": "UDP port number",
     "type": "string",
     "pattern": "^([1-9][0-9]{0,3}|[1-5][0-9]{4}|6[0-4][0-9]{3}|65[0-4][0-9]{2}|655[0-2][0-9]|6553[0-5])$"
+  },
+  "ContentPair": {
+    "title": "content_pair",
+    "type": "object",
+    "properties": {
+      "contentKey": {
+        "title": "Content key",
+        "description": "The encoded Portal content key",
+        "$ref": "#/components/schemas/hexString"
+      },
+      "contentValue": {
+        "title": "Content value",
+        "description": "The encoded Portal content value",
+        "$ref": "#/components/schemas/hexString"
```

Comment:
ContentKey and ContentValue already exist here: https://github.com/ethereum/portal-network-specs/blob/master/jsonrpc/src/content/params.json#L2-L16

I guess they could be re-used here inside the `ContentPair` (or `ContentItem`).

Reply:
I'm curious about the choice for `ContentPair` to be an **object**, rather than a **tuple** `[key, value]`.

Reply:
I didn't know tuple was a type in openapi I will updated it

Reply:
> I'm curious about the choice for `ContentPair` to be an **object**, rather than a **tuple** `[key, value]`.

I couldn't find a tuple type so I ended up using an array to achieve a similar thing

Reply:
> ContentKey and ContentValue already exist here: https://github.com/ethereum/portal-network-specs/blob/master/jsonrpc/src/content/params.json#L2-L16
> 
> I guess they could be re-used here inside the `ContentPair` (or `ContentItem`).

I am not sure how to do this, I tried a few times. Maybe someone else knows how?